README.md

```diff
@@ -191,6 +193,15 @@ account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_en
 You can also enable this per-service account with the annotation
 `eks.amazonaws.com/sts-regional-endpoints` set to `"true"`.
 
+### pod-identity-webhook ConfigMap
```

Comment:
Thanks for adding this.  I think I'd like to see two things, a short example of the configmap contents, and a note/link that this was added for kops and similar use cases (I can imagine it might be a bit user unfriendly to add the configmap and update it by hand).

Reply:
Thought of that, but felt it may be a bit verbose. Added now.

Reply:
Thanks.  We can always move to secondary readme page to avoid verbosity but I think its good to have it documented.